modules/ROOT/pages/secure-application-properties.adoc

```diff
@@ -192,26 +195,32 @@ Then, at deployment time, the operator can type in the `secure.key` value into t
 
-But there is a way for a Mule application to be designed to allow operations staff to safely override encrypted properties.
-If encrypted property names are also listed in the Mule application's 'secureProperties' entry in 'mule-artifact.json', then once the application is deployed, the value of this encrypted property will also be hidden in the Runtime Manager Properties tab, just like decrypted properties, and just like other properties, the decrypted value is only stored in the Mule worker's memory and securely stored in the CloudHub database for your user account, and is never stored in any CloudHub worker's files, nor is it ever passed between any other machines (including the Runtime Manager Console).
+If encrypted property names are listed in the Mule application's `secureProperties` entry in `mule-artifact.json`, once the application is deployed, the value of the encrypted property names is hidden in the Runtime Manager *Properties* tab, along with decrypted properties. The decrypted values are stored securely in the Mule worker's memory and in the CloudHub database for your user account. These decrypted values are never stored in any CloudHub worker's files, nor are they ever passed between any other machines (including the Runtime Manager Console).
 
-So for any encrypted property that is also marked as safely hidden in the Mule application's `secureProperties` entry, you can safely replace any encrypted property with a clear-text value, which will be securely stored in the CloudHub properties database, and securely passed in to the Mule application ever time the Mule application is started.
+So for any encrypted property that is also marked as hidden in the Mule application's `secureProperties` entry, you can safely replace any encrypted property with a clear-text value. The clear-text value is then securely stored in the CloudHub properties database, and securely passed into the Mule application every time the Mule application is started.
 
-The new value is also hidden once the application is deployed and can never be viewed again.
+The new value is hidden once the application is deployed and can never be viewed again.
 
-This means that when you override a secure property in the CloudHub properties tab for the application, the values never needs to be encrypted. In this scenario, securing the values of sensitive properties is reduced to controlling which operators have access to those values when they deploy or redeploy the secure application.
+This means that when you override a secure property in the CloudHub *Properties* tab for the application, the value never needs to be encrypted. In this scenario, only users who have access to the values when they deploy or redeploy a secure application can override secure properties. 
```

Comment:
This what? Do we mean Thus or Therefore? Also (okay to punt), I'm not really sure what we're saying with the second sentence. What does it mean to have access to the values? Are we saying anyone with permission to edit can override any secure property (but they can't view it once deployed)?